components/Ripple/src/MDCRippleTouchController.h

```diff
@@ -107,4 +135,19 @@
          didProcessRippleView:(nonnull MDCRippleView *)rippleView
               atTouchLocation:(CGPoint)location;
 
+/**
+ Inserts the ripple view into the given view.
+
+ If this method is not implemented, the ripple view is added as a subview of the given view provided
+ in the controller's `addRippleToView:` method or convenience initializer `initWithView:`.
+ Delegates can choose to insert the ripple view anywhere in the view hierarchy.
+
+ @param rippleTouchController The ripple touch controller.
+ @param rippleView The ripple view.
+ @param view The view to add the ripple view to.
+ */
+- (void)rippleTouchController:(nonnull MDCRippleTouchController *)rippleTouchController
+             insertRippleView:(nonnull UIView *)rippleView
```

Comment:
`insertRippleView` should be of type `MDCRippleView *` to match the other API in the delegate.

Reply:
Done